R/magimage.R

```diff
@@ -0,0 +1,42 @@
+magimage<-function(mat, astr=NULL, main="", xlab=ifelse(is.null(astr),"X (pix)","RA (deg)"), ylab=ifelse(is.null(astr),"Y (pix)","Dec (deg)"), side=1:4,labels=c(T,T,F,F),...) {
```

Comment:
I'd move "astr" much further down the paramter list I think- it's quite unfamiliar to people who are using this instead of base image.